### PR TITLE
fix integer to bigint for extensions

### PR DESCRIPTION
**Pull request by ChuckNorrison**
Some more INTEGER to BIGINT conversion for postgresql database

### CODE REVIEW

**Comment by callebtc** [comment]
LGTM. Tests need fix, probably unrelated to the PR?

**Review by callebtc — APPROVED** [review]

File: lnbits/extensions/satspay/migrations.py [hunk 1]
```diff
@@ -18,8 +18,8 @@ async def m001_initial(db):
             completelink TEXT,
             completelinktext TEXT,
             time INTEGER,
-            amount INTEGER,
-            balance INTEGER DEFAULT 0,
+            amount {db.big_int},
```
Comment (motorina0) [inline lnbits/extensions/satspay/migrations.py:21]: must use `f"""...`

File: lnbits/extensions/watchonly/migrations.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ async def m001_initial(db):
             masterpub TEXT NOT NULL,
             title TEXT NOT NULL,
             address_no INTEGER NOT NULL DEFAULT 0,
-            balance INTEGER NOT NULL
+            balance {db.big_int} NOT NULL
```
Comment (motorina0) [inline lnbits/extensions/watchonly/migrations.py:13]: must use `f"""...`

File: lnbits/extensions/watchonly/migrations.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ async def m001_initial(db):
             id TEXT NOT NULL PRIMARY KEY,
             address TEXT NOT NULL,
             wallet TEXT NOT NULL,
-            amount INTEGER NOT NULL
+            amount {db.big_int} NOT NULL
```
Comment (motorina0) [inline lnbits/extensions/watchonly/migrations.py:24]: must use `f"""...`

**Comment by motorina0** [comment]
I've re-triggered the checks. It looks like tests fail now.

File: lnbits/extensions/satspay/migrations.py (hunk 1 above)
Comment (ChuckNorrison) [reply lnbits/extensions/satspay/migrations.py:21]: it fails with `sqlite3.OperationalError unrecognized token: "{"`. looks like the f is missing here.

**Review by motorina0 — APPROVED** [review]
LGTM

**Comment by callebtc** [comment]
Thank you for this awesome PR, this bug has been bugging for so long!

**Pull request merged**